src/chaos-experiments/models/kubernetes/chaosExperimentConnector.js

```diff
@@ -0,0 +1,119 @@
+const util = require('util');
+const fs = require('fs');
+const kubernetesConfig = require('../../../config/kubernetesConfig');
+const logger = require('../../../common/logger');
+const requestSender = require('../../../common/requestSender');
+const { CHAOS_EXPERIMENT_LABELS, PREDATOR_RUNNER_PREFIX } = require('../../../../src/common/consts');
+const kubernetesUrl = kubernetesConfig.kubernetesUrl;
+
+const TOKEN_PATH = '/var/run/secrets/kubernetes.io/serviceaccount/token';
+const headers = {};
+const { APP, JOB_ID } = CHAOS_EXPERIMENT_LABELS;
+const STATUS_TO_CLEAN = 'Stop';
+let supportedChaosKinds;
+
+if (kubernetesConfig.kubernetesToken) {
```

Comment:
consider doing this in an `init` function and add to the init function the GET supported kinds

Reply:
done